### PR TITLE
tests: refine coverage for embedded players

### PR DESCRIPTION
**Pull request by catch-21**
- refine iframe coverage
- more coverage for ProviderYoutube

### CODE REVIEW

File: src/components/molecules/PostLinkEmbeds/Providers/Youtube/ProviderYoutube.test.ts [hunk 1]
```diff
@@ -276,5 +353,25 @@ describe('ProviderYoutube', () => {
         });
       });
     });
+
+    describe('case sensitivity and domain variations', () => {
+      // Fails because url is not normalized to lowercase before parsing
+      // Doing so doesn't appear to break the video ID, but could it?
+      it.skip('handles uppercase domains gracefully', () => {
```
[the remark this answers is not in the document]
Comment (secondl1ght) [reply src/components/molecules/PostLinkEmbeds/Providers/Youtube/ProviderYoutube.test.ts]: It's uncommon to see uppercase URLs, but we could handle this edge case... I am pretty sure the web standard is that all parts of a URL are supposed to be case-insensitive (and seems that YT is following this pattern). If making small changes handles these comments we could do it, it's nice to have as much coverage as possible, but we also don't want to get too far in the weeds covering cases that are super rare with no downsides. We could take the approach to handle things like this if they come up from real users of the app.

**Review by MiguelMedeiros — APPROVED** [review]

**Comment by MiguelMedeiros** [comment]
Enabled both tested and fixed the uppercase.

File: src/components/atoms/Iframe/Iframe.test.tsx [hunk 2]
```diff
@@ -75,9 +84,11 @@ describe('Iframe - Snapshots', () => {
       <Iframe
         src="https://example.com/embed"
         title="Example embed"
-        width="100%"
-        height="315"
-        data-testid="test-iframe"
+        className="custom-class"
+        width="640"
```
Comment (catch-21) [inline src/components/atoms/Iframe/Iframe.test.tsx]: The `width=100%` and `height=315` set previously was done so to match the implementation.
